pom.xml

```diff
@@ -447,6 +447,12 @@
 				<type>jar</type>
 			</dependency>
 			<dependency>
+		    	<groupId>com.tdunning</groupId>
+		    	<artifactId>json</artifactId>
+				<version>1.2-SNAPSHOT</version>
```

Comment:
@klopfdreh 
I'm done with wicketstuff. I think 4-5 modules (select2, whiteboard, etc...) should be manually tested but don't expect problems.